src/Sylius/Behat/Context/Ui/CheckoutContext.php

```diff
@@ -527,31 +517,20 @@ public function iSignIn()
      */
     public function iShouldSeeTheThankYouPage()
     {
-        /** @var UserInterface $user */
-        $user = $this->sharedStorage->get('user');
-        $customer = $user->getCustomer();
-
-        expect($this->checkoutThankYouPage->hasThankYouMessageFor($customer->getFullName()))->toBe(true);
+        Assert::true(
+            $this->thankYouPage->hasThankYouMessage(),
+            'I should see thank you message, but I do not'
+        );
     }
 
     /**
      * @Then I should be redirected back to the thank you page
      */
     public function iShouldBeRedirectedBackToTheThankYouPage()
     {
-        $this->checkoutThankYouPage->waitForResponse(5);
-
-        expect($this->checkoutThankYouPage->isOpen())->toBe(true);
-    }
-
-    /**
-     * @Then I should be redirected back to the order payment page
-     */
-    public function iShouldBeRedirectedBackToTheOrderPaymentPage()
-    {
-        $this->orderPaymentsPage->waitForResponse(5, ['number' => $this->getLastOrder()->getNumber()]);
+        $this->thankYouPage->waitForResponse(5);
 
-        expect($this->orderPaymentsPage->isOpen(['number' => $this->getLastOrder()->getNumber()]))->toBe(true);
+        expect($this->thankYouPage->isOpen())->toBe(true);
```

Comment:
We don't use `expect` anymore. Use `Assert::true($this->thankYouPage->isOpen())`